libs/platform/i18n/src/lib/models/text.model.ts

```diff
@@ -1,5 +1,9 @@
 import { I18nContext } from '@spryker-oryx/utilities';
 
+/**
+ * @deprecated since 1.4. Use I18nContent interface
```

Comment:
why not use TextResource if we already have it? 🤔

Reply:
1) Due to managing of the dependancies. For current case ui domain (notification center) should not import from platform 
2) It will be still related to i18n, but to the its utilities part, and will be much more logical and convenient to keep it together with directive